extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/OidcRequestContextProperties.java

```diff
@@ -19,22 +19,55 @@ public OidcRequestContextProperties(Map<String, Object> properties) {
         this.properties = properties;
     }
 
+    /**
+     * Get property value
+     *
+     * @param name property name
+     * @return property value
+     */
     public <T> T get(String name) {
         @SuppressWarnings("unchecked")
         T value = (T) properties.get(name);
         return value;
     }
 
+    /**
+     * Get property value as String
+     *
+     * @param name property name
+     * @return property value as String
+     */
     public String getString(String name) {
         return (String) get(name);
     }
 
+    /**
+     * Get typed property value
+     *
+     * @param name property name
+     * @param type property type
+     * @return typed property value
+     */
     public <T> T get(String name, Class<T> type) {
         return type.cast(get(name));
     }
 
+    /**
+     * Get an unmodifiable view of the current context properties.
+     *
+     * @return all properties
+     */
     public Map<String, Object> getAll() {
         return Collections.unmodifiableMap(properties);
     }
 
+    /**
+     * Set the property
+     *
+     * @param name property name
+     * @param value property value
+     */
+    public void put(String name, Object value) {
```

Comment:
How about returning `this` to enable method chaining?

```suggestion
    public OidcRequestContextProperties put(String name, Object value) {
```

Reply:
@gastaldi Done, thanks

Reply:
Sorry for the noise, just added some `return` description